recipes/cfgfile/0.2.8.2.1/conanfile.py

```diff
@@ -0,0 +1,66 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class CfgfileConan(ConanFile):
+    name = "cfgfile"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/igormironchik/cfgfile.git"
+    license = "MIT"
+    description = "Header-only library for reading/saving configuration files with schema defined in sources."
+    generators = "cmake"
+    topics = ("conan", "cfgfile", "configuration")
+    settings = "os", "arch", "compiler", "build_type"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "Visual Studio": "15",
+            "gcc": "5",
+            "clang": "3.5",
+            "apple-clang": "10"
+        }
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.configure(source_folder=os.path.join(self._source_subfolder, "generator"), build_folder=self._build_subfolder)
```

Comment:
It requires CMake wrapper.

Reply:
Could you specify what do you mean under CMake wrapper? I guess that I already use it.

Reply:
https://github.com/conan-io/conan-center-index/blob/master/recipes/acado/all/CMakeLists.txt

Is an example